app/scripts/angular-apimock.js

```diff
@@ -77,38 +99,31 @@ angular.module('apiMock', [])
 			return keys;
 		}
 
-		// Taken from Angular 1.4.x: https://github.com/angular/angular.js/blob/929ec6ba5a60e926654583033a90aebe716123c0/src/ng/http.js#L18
+		// Modified from Angular 1.4.x: https://github.com/angular/angular.js/blob/929ec6ba5a60e926654583033a90aebe716123c0/src/ng/http.js#L18
 		function serializeValue(v) {
-			if (angular.isObject(v)) {
-				return angular.isDate(v) ? v.toISOString() : angular.toJson(v);
+			if (angular.isDate(v)) {
+				return toISOString(v);
 			}
+
```

Comment:
The cases don't get more covered with a ternary operator, compared to a regular if/guard clause? It would be a one-liner which might be nicer, but it won't cover anything different?